docs/README.md

```diff
@@ -12,7 +12,7 @@ You can grab the latest nightly binary executable for:
 - [Linux](https://storage.googleapis.com/knative-nightly/client/latest/kn-linux-amd64)
 - [Windows](https://storage.googleapis.com/knative-nightly/client/latest/kn-windows-amd64.exe)
 
-Put it on your system path, and make sure it's executable.
+Please put it on your system path, and make sure it's executable.
```

Comment:
Don't add "please" in technical docs, this is actually called out in the Google developer style guide.